ui/src/components/ExtensionCard.vue

```diff
@@ -2,18 +2,10 @@
   <div class="card">
     <div class="tags px-6 py-4">
       <div>
-        <span class="font-semibold">RowKey:</span>
-        <span class="extension-tag">{{ resource.RowKey }}</span>
+        <span class="font-semibold"><a :href="resource.url" target="_blank" rel="noopener noreferrer">{{resource.title}}</a></span>
       </div>
       <div>
-        <span class="font-semibold">Timestamp:</span>
-        <span class="extension-tag">{{ resource.Timestamp }}</span>
-      </div>
-      <div>
-        <span class="font-semibold">URL:</span>
-        <a :href="resource.url"
-          ><span class="extension-tag">{{ resource.url }}</span></a
-        >
+        <span class="extension-tag">{{ resource.description }}</span>

```

Comment:
```suggestion
        <p>{{ resource.description }}</p>

```

The `extension-tag` class is used to style the metadata tags for a resource.